DownFall/Scenes/GameScene.swift

```diff
@@ -41,6 +41,10 @@ class GameScene: SKScene {
     //level
     private var level: Level?
     
+    // targeting
+    private var targetingViewModel: TargetingViewModel?
```

Comment:
I dont think we need a reference to this, as long as we have one strong reference between wm and wiemodel we should be good